ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+#!/apps/bio/software/anaconda2/envs/hcp/bin/python3
```

Comment:
Do we need a shebang, or should this rather be set by having a conda environment?

Reply:
It seems like quite a cumbersome thing to do, loading a conda env just to execute a cronscript, as we are going to upgrade the systems to RHEL8 and use python3 as standard. I just stole the path from the original script to avoid imports and simplify future execution. I would try to stay away from python2 as much as possible.. Python2 is dead(ish).. long live Python3.